eng/pipelines/coreclr/templates/superpmi-collect-job.yml

```diff
@@ -33,7 +37,16 @@ jobs:
      - ${{ if eq(parameters.collectionName, 'coreclr_tests') }}:
         - 'coreclr_common_test_build_p1_AnyOS_AnyCPU_${{parameters.buildConfig }}'
 
-    variables: ${{ parameters.variables }}
+    variables:
+
+    - name: crossArg
+      value: ''
+    - ${{ if eq(parameters.crossBuild, true) }}:
+      - name: crossArg
+        value: '-cross'
```

Comment:
why was this not problem until now? did something changed lately?

Reply:
agocke wrote: "I made a change to AOT the xunit log checker during the build that could be affected by linker configuration. Ensure that your pipeline is passing “-cross” during the build — that should force usage of lld instead of bfd"

This led to https://github.com/dotnet/runtime/pull/107229, for example